addons/sale/sale.py

```diff
@@ -652,7 +652,7 @@ def write(self, values):
     price_unit = fields.Float('Unit Price', required=True, digits=dp.get_precision('Product Price'), default=0.0)
 
     price_subtotal = fields.Monetary(compute='_compute_amount', string='Subtotal', readonly=True, store=True)
-    price_tax = fields.Monetary(compute='_compute_amount', string='Taxes', readonly=True, store=True)
+    price_tax = fields.Float(compute='_compute_amount', digits=(16, 7), string='Taxes', readonly=True, store=True)
```

Comment:
This changes the DB layout, so this is not a valid solution. You have to recalculate the taxes of the lines in case of global round or find another method.